docs/src/documentation/03-components/04-overlay/dialog/03-accessibility.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/dialog/accessibility/
+---
+
+## Accessibility
+
+The Dialog component has been designed with accessibility in mind, providing features that enhance the experience for users of assistive technologies.
+
+### Accessibility props
+
+The following props are available to improve the accessibility of your Dialog component:
+
+| Name        | Type                                                    | Description                                                                                                                     |
+| :---------- | :------------------------------------------------------ | :------------------------------------------------------------------------------------------------------------------------------ |
+| title       | `React.Node`                                            | Provides a visible title that is associated with the dialog as its accessible name.                                             |
+| titleAs     | `"h1" \| "h2" \| "h3" \| "h4" \| "h5" \| "h6" \| "div"` | Defines the semantic HTML element for the title while maintaining its visual appearance.                                        |
+| description | `React.Node`                                            | Provides additional context about the dialog's purpose, which is automatically associated with the dialog via aria-describedby. |
+| onClose     | `() => void \| Promise`                                 | Callback that is triggered when the dialog is closed, supporting keyboard interaction and screen readers.                       |
```

Comment:
> and screen readers.  

What does this mean in this context?

Reply:
Looking at it, I think the whole `onClose` prop can be removed from here.